packages/common/src/utils/formatters.ts

```diff
@@ -1,6 +1,11 @@
 import format from 'date-fns/format';
 import isValid from 'date-fns/isValid';
 
+// tax as a number like 12 for 12%
+// TODO: Do something better than naively rounding to 2 decimal places
+export const formatTax = (tax: number, withParens = true) =>
```

Comment:
there's a couple of places where I'm seeing unrounded values.. e.g.

Add service charge modal:
<img width="307" alt="Screen Shot 2022-03-14 at 11 14 16 AM" src="https://user-images.githubusercontent.com/9192912/158081990-64fd7f76-3ab0-4d6d-8771-0235a9fd96c3.png">

Edit tax rate modal:
<img width="999" alt="Screen Shot 2022-03-14 at 11 14 37 AM" src="https://user-images.githubusercontent.com/9192912/158081993-3db919e3-d835-4254-b32d-2fdda37ec648.png">

Can the value be rounded for display while still using a float for the value?

maybe the rounding can be addressed when we get prefs? there's a few relating to rounding

Reply:
Yes I wasn't sure what to round to. I think these are rounding to 20 decimal places..? At least the currency formatter is..

I've no idea to what precision they should be shown!